docs/tooling/kubefirst-cli.md

```diff
@@ -0,0 +1,148 @@
+# Kubefirst Command Line
+
+Kubefirst provides a CLI to empower the whole previsioning cluster management process. The basic usage of the CLI is to
+fully create the Kubefirst previsioning process, but there are also many other handful options to backup, validation and
+cluster destroy.
+
+## Kubefirst CLI available commands
+
+- [clean](#clean)
+- [init](#init)
+- [cluster](#cluster-create)
+- [destroy](#destroy)
+- [backupSSL](#backupssl)
+- [restoreSSL](#restoressl)
+- [state](#state)
+
+### clean
+
+Kubefirst creates files, folders and cloud buckets during installation at your environment. This command removes and
+re-create Kubefirst base files. To destroy cloud resources you need to specify additional flags (--destroy-buckets)
+
+```
+Usage:
+kubefirst clean [flags]
+
+Flags:
+--destroy-buckets   destroy buckets created by init cmd
+--destroy-confirm   confirm destroy operation (to be used during automation process)
+-h, --help              help for clean
+--rm-logs           remove logs folder
+```
+
+### init
+
+Init command will prepare the installation, and create the initial resources that will support the installation when 
+`kubefirst cluster create` is be called.
+
+```
+Usage:
+kubefirst init [flags]
+
+Flags:
+--admin-email string        the email address for the administrator as well as for lets-encrypt certificate emails
+--aws-assume-role string    instead of using AWS IAM user credentials, AWS AssumeRole feature generate role based credentials, more at https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html
+--aws-nodes-spot            nodes spot on AWS EKS compute nodes
+--clean                     delete any local kubefirst content ~/.kubefirst, ~/.k1
+--cloud string              the cloud to provision infrastructure in
+--cluster-name string       the cluster name, used to identify resources on cloud provider (default "kubefirst")
+--dry-run                   set to dry-run mode, no changes done on cloud provider selected
+--github-host string        Github URL (default "github.com")
+--github-org string         Github Org of repos
+--github-owner string       Github owner of repos
+--github-user string        Github user
+--gitops-branch string      version/branch used on git clone - former: version-gitops flag
+--gitops-owner string       git owner of gitops, this may be a user or a org to support forks for testing (default "kubefirst")
```

Comment:
```suggestion
--gitops-owner string       Git owner of GitOps - this may be a user or an org to support forks for testing (default "kubefirst").
```